Tests/js/models/assets/ez-usermodel-tests.js

```diff
@@ -40,11 +54,37 @@ YUI.add('ez-usermodel-tests', function (Y) {
         tearDown: function () {
             this.model.destroy();
             delete this.model;
-        }
+        },
+
+        "Sync 'read' should load the content with CAPI": function () {
+            var m = this.model,
+                modelId = "/api/v2/ezp/model/mid",
+                callback = function () { };
+
+            Y.Mock.expect(this.capiMock, {
+                method: this.capiGetService,
+                returns: this.serviceMock
+            });
+            Y.Mock.expect(this.serviceMock, {
+                method: this.serviceLoad,
+                args: [
+                    modelId,
+                    Y.Mock.Value.Function
```

Comment:
since you don't call this function, the entire `loadUser` callback is not covered. This mock shoud be configured to run that function something like:
```
Mock.expect(this.serviceMock, {
   method: this.serviceLoad,
   args: [modelId, Mock.Value.Function],
   run: Y.bind(function (id, callback) {
       callback(false, this.loadResponse);
   }, this)
});
```
this will have several consequences:
1. the configured `loadResponse` should have a `User.ContentType._href` value
1. you should dynamically create a fake version of eZ.ContentType
1. to get full coverage the mock should also be configured in another test to execute the callback with a truthy value as the error